make/config.mk

```diff
@@ -55,7 +55,7 @@ export DEBUG
 
 
 OPT	:=							\
-	$(if $(filter release,$(LRSTAR_BUILD_TYPE)),-Os,-Og)	\
+	$(if $(filter release,$(LRSTAR_BUILD_TYPE)),-Wall -Os,-Wall -Og)	\
```

Comment:
Thanks for doing this! I've been meaning to enable a higher level of diagnostics on lrstar and dfa, but had not gotten to work on it yet. 

Enabling '-Werror' is also highly desirable.

Reply:
That's true but still there is some warnings pending.